yaksh/views.py

```diff
@@ -284,9 +284,11 @@ def prof_manage(request, msg=None):
     user = request.user
     ci = RequestContext(request)
     if user.is_authenticated() and is_moderator(user):
-        question_papers = QuestionPaper.objects.filter(quiz__course__creator=user,
-                                                       quiz__is_trial=False
-                                                       )
+        question_papers = QuestionPaper.objects.filter(
+                                                Q(quiz__course__creator=user) |
+                                                Q(quiz__course__teachers=user),
+                                                quiz__is_trial=False
+                                                )
```

Comment:
Are you sure this will not return duplicate question papers?
Since you are using ```Q```, and a teacher can be in multiple courses.